config/.rubocop.yml

```diff
@@ -7,7 +7,7 @@ AllCops:
     - 'Vagrantfile'
 
 # Multi-line method chaining should be done with leading dots.
-Layout/DotPosition:
+Metrics/DotPosition:
```

Comment:
Please revert the changes in this file. (We can discuss these changes separately, but not in this PR.)